packages/legend-application-studio/src/components/editor/side-bar/Explorer.tsx

```diff
@@ -283,20 +284,44 @@ const ExplorerContextMenu = observer(
     };
     const copyWorkspaceElementLink = (): void => {
```

Comment:
can we add a `copy SDLC link` option as well to copy the sdlc project view in addition to this one. 
Thanks